.obs/workflows.yml

```diff
@@ -73,36 +73,41 @@ pr_workflow:
         target_project: isv:Rancher:Elemental:PR
     - branch_package:
         source_project: isv:Rancher:Elemental:Dev
-        source_package: build-iso-base-image
+        source_package: build-base-iso-image
         target_project: isv:Rancher:Elemental:PR
     - branch_package:
         source_project: isv:Rancher:Elemental:Dev
-        source_package: build-iso-kvm-image
+        source_package: build-kvm-iso-image
         target_project: isv:Rancher:Elemental:PR
     - branch_package:
         source_project: isv:Rancher:Elemental:Dev
-        source_package: build-iso-rt-image
+        source_package: build-rt-iso-image
         target_project: isv:Rancher:Elemental:PR
     - branch_package:
         source_project: isv:Rancher:Elemental:Dev
-        source_package: build-iso-image
+        source_package: build-baremetal-iso-image
         target_project: isv:Rancher:Elemental:PR
     - branch_package:
         source_project: isv:Rancher:Elemental:Dev
-        source_package: SLE-Micro-container
+        source_package: SL-Micro-baremetal-container
         target_project: isv:Rancher:Elemental:PR
     - branch_package:
         source_project: isv:Rancher:Elemental:Dev
-        source_package: SLE-Micro-base-container
+        source_package: SL-Micro-base-container
         target_project: isv:Rancher:Elemental:PR
     - branch_package:
         source_project: isv:Rancher:Elemental:Dev
-        source_package: SLE-Micro-kvm-container
+        source_package: SL-Micro-kvm-container
         target_project: isv:Rancher:Elemental:PR
     - branch_package:
         source_project: isv:Rancher:Elemental:Dev
-        source_package: SLE-Micro-rt-container
+        source_package: SL-Micro-rt-container
         target_project: isv:Rancher:Elemental:PR
+    - set_flags:
+        flags:
+          - type: publish
+            status: enable
+            project: isv:Rancher:Elemental:PR
```

Comment:
Lets try to automatically publish the images for PRs, if https://github.com/openSUSE/open-build-service/issues/14978 is someday fixed that will be useful.